sdk/test-utils/recorder/src/baseRecorder.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+import { handleSingleQuotesInUrlPath } from "./utils/index";
```

Comment:
reason for this change? it feels to me that this line could be merged into `from "./utils"`?

Reply:
I was resolving a bad merge and had this when we had both `utils/index.ts` and `utils.ts`.
Now that I've deleted `utils.ts`, this can be moved.

Reply:
Done!